SimPEG/EM/TDEM/SrcTDEM.py

```diff
@@ -310,36 +313,24 @@ class MagDipole(BaseTDEMSrc):
     orientation = properties.Vector3(
```

Comment:
not sure it seems previously only 'Z' was working. 
now it's fine for all three directions. if we want to put arbitrary direction, 
can we do np.array([0.1, 0.1, 0.5])?

Reply:
cool!